python/tvm/relay/frontend/pytorch.py

```diff
@@ -1141,14 +1264,14 @@ def _impl(inputs, input_types):
             bias = inputs[0]
             return _op.nn.bias_add(dense_out, bias)
         else:
-            return dense_out
+            return dense_out + _expr.const(inputs[0])
```

Comment:
why this is needed? This is for use_bias == False case right?

Reply:
@kevinthesun Given that this `def _dense()` is for converting `aten::addmm`, can you rename this converter to `_addmm` and update variable names according to https://pytorch.org/docs/stable/generated/torch.addmm.html? In particular, please remove `use_bias` as this name doesn't make any sense for `addmm` op.

This code comes from the original PR, we need to remove technical debt as much as possible...